aviatrix/resource_aviatrix_edge_gateway_selfmanaged.go

```diff
@@ -887,3 +915,37 @@ func resourceAviatrixEdgeGatewaySelfmanagedDelete(ctx context.Context, d *schema
 
 	return nil
 }
+
+func editAdvertisedSpokeRoutesWithRetry(client *goaviatrix.Client, gatewayForGatewayFunctions *goaviatrix.Gateway, d *schema.ResourceData) error {
+	const maxRetries = 30
+	const retryDelay = 10 * time.Second
+	includedAdvertisedSpokeRoutes := getStringList(d, "included_advertised_spoke_routes")
+	if len(includedAdvertisedSpokeRoutes) > 0 {
```

Comment:
```suggestion
	if len(includedAdvertisedSpokeRoutes) > 0 {
		return nil
	}
```

If you return early, then you can reduce the indent of the main work of the function.

https://go.dev/wiki/CodeReviewComments#indent-error-flow